packages/@orchidui-vue/src/Form/Input/OcInput.vue

```diff
@@ -146,7 +155,7 @@ const inputClasses = computed(() => [
               isFocused = false;
               $emit('blur');
             "
-            @input="$emit('update:modelValue', $event.target.value)"
+            @input="$emit('update:modelValue', updateValue($event.target.value))"
```

Comment:
not sure the logic here. Here we format value before emit, but the `value` is formatted 1 one time in line 143.

Reply:
Fixed. Replaced computed  by ref.